apps/nextjs-example/src/app/page.tsx

```diff
@@ -246,13 +247,13 @@ function WalletConnection({
                 subLabel: "(only if attached)",
                 value: <p>{account?.ansName ?? "Not Present"}</p>,
               },
-              {
-                label: "Min keys required",
-                subLabel: "(only for multisig)",
-                value: (
-                  <p>{account?.minKeysRequired?.toString() ?? "Not Present"}</p>
-                ),
-              },
+              // {
+              //   label: "Min keys required",
+              //   subLabel: "(only for multisig)",
+              //   value: (
+              //     <p>{account?.minKeysRequired?.toString() ?? "Not Present"}</p>
+              //   ),
+              // },
```

Comment:
Is this not working at the moment?

Reply:
No, the standard does not consider multi sig